src/js/components/TextArea/TextArea.js

```diff
@@ -15,7 +15,7 @@ const TextArea = ({ fill, forwardRef, ...rest }) => {
   };
 
   return (
-    <Keyboard onEsc={onEsc}>
+    <Keyboard onEsc={onEsc} onKeyDown={rest.onKeyDown}>
```

Comment:
It doesn't look correct to pass `rest.onKeyDown` to both `Keyboard` and to `StyledTextArea`. I'm wondering why the textarea isn't delivering the onKeyDown event since the handler is getting passed via `{...rest}`?

Reply:
Thanks @ericsoderberghp, I'll admit this was a pretty optimistic PR. I'll do some more investigation if I get time.